tardis/energy_input/gamma_ray_packet_source.py

```diff
@@ -737,36 +828,36 @@ def create_packets(
 
         # sample radii at time = 0
         initial_radii = self.create_packet_radii(sampled_packets_df)
+        # sample decay times
+        sampled_times = (
+            sampled_packets_df.index.get_level_values("time") * 86400.0
```

Comment:
This could be a constant defined before called DAYS_TO_SECONDS or something like that

Reply:
Please address this comment @Knights-Templars